ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -53,6 +55,43 @@ if [[ $PUBLIC_VLAN == "false" ]]; then
   echo -e "controlplane_network: 192.168.216.1/21\ncontrolplane_network_prefix: 21" >> /tmp/all.yml
 fi
 
+if [[ ${TYPE} == 'vmno' ]]; then
```

Comment:
Why having two different approaches to deploy hybrid clusters?
I like the idea of the vmno type btw, but we need to consolidate on one.
cc @mcornea

Reply:
Gotcha, makes sense.
Never the less, to use the `TYPE` field is interesting because with one look you can see which kind of deployment are we looking to, i.e.: we could have `mno`, `sno`, `hybrid` and `vmno` (even `compact` maybe). All this from a Prow perspective I mean.